examples/water_SCAN/ice04/command.sh

```
@@ -0,0 +1,37 @@
+# docker pull yfb222333/dpti-lammps-fep:latest
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)